### ISSUE TITLE
Output colors is not same as tsc --watch

### ISSUE THREAD

**Issue by jonaskello** [gilamran/tsc-watch#15]
This seems like a nice lib and I will give it a try :-).

I'm using vscode integrated terminal with solarized light settings. The colors of the output is not the same of regular `tsc --watch` and `tsc-watch`, see below. The colors output by `tsc-watch` is very hard to read with my settings.

```
$ yarn tsc --watch
```

![image](https://user-images.githubusercontent.com/664076/35474092-8757cda0-0389-11e8-89d5-bd75df9c39e6.png)

```
$ yarn tsc-watch
```

![image](https://user-images.githubusercontent.com/664076/35474099-a5e80bb8-0389-11e8-9657-f229c76ec4e7.png)

**Comment by gilamran** [comment]
I don't have this problem... 
What is your setup? Where are you running this?
I'll need more information and tests

**Comment by jonaskello** [comment]
I'm using vscode 1.19.2 on MacOS 10.12.4. I have the theme set to "Solarized light" within vscode and I am using the vscode integrated terminal to run the commands.

**Comment by jonaskello** [comment]
My guess is that the problem is [here](https://github.com/gilamran/tsc-watch/blob/master/lib/tsc-watch.js#L36) where tsc-watch hardcodes the default color to white. I think tsc in original form adheres more to the colors of the environment.

**Comment by gilamran** [comment]
@jonaskello Cool, nice catch!
I'm also using vccode and I just switched to "Solatized light", run tsc-watch on the integrated terminal and I don't see the issue... Maybe because I'm using Windows and the integrated terminal isn't the same as Mac.
I'm guessing that you're probably right about the environment color... Maybe you can try on your side and PR?

**Comment by jonaskello** [comment]
I tried it on my Windows machine now which also has vscode with solarized light. You're right, the issue does not appear there. I'll see if I can find a way to make it use the environment colors.

There is a related issue which isn't really that important but I just noticed it now. If you do `tsc-watch --help` the output will be colorized to green at the point where it prints help about the `-w` parameter. I guess this is because that part is matched by the regex for green color. This happens on both Mac and Windows.
